### ISSUE TITLE
Devil trigger meter stuck at 5 bars

### ISSUE THREAD

**Issue by AceDecimation** [serpentiem/ddmk#306]
Uploading Devil May Cry HD Collection_2022.08.02-21.40.mp4…

It became like this when i bought the 7th Purple orb, it was normal when i was at 6 orbs

**Comment by nosh62** [comment]
The video isn't embedded

**Comment by AceDecimation** [comment]
> The video isn't embedded

Oh shit sorry

https://user-images.githubusercontent.com/82496438/182606156-ce28936f-5efb-4d38-88e3-4ad44cfe6856.mp4

**Comment by nosh62** [comment]
Have you checked if this bug happens when you don't use that HUD? Also, try deleting DDMK and buying the D.T. upgrades with the default menu. Be sure to make a backup of the DDMK files.

**Comment by AceDecimation** [comment]
uhh no, let me try that

**Comment by AceDecimation** [comment]
Yeah did it, its still the same even updated ddmk

**Comment by nosh62** [comment]
Did you 100& remove DDMK from the DMC HDC folder? You need to remove **dinput8.dll**, **Eva.dll**, **Lucia.dll**, and **Mary.dll**, the **images** folder, and the **GData.afs** folder. Everything but the **GData.afs** folder is on the DMC HDC folder. Hopefully, this will work. I don't know how this specific code works, just how general codes works

**Comment by AceDecimation** [comment]
Yes, i even made sure and deleted everyfile and did the "check file integrety " thing

**Comment by nosh62** [comment]
Then the only option I can think of is to completely delete DDMK and start all over (talking about the styles' level). Other than that, I don't know how to help ya. Sorry
